tests/testthat/test-winsorization.R

```diff
@@ -11,11 +11,11 @@ test_that("with missing values", {
 test_that("winsorize: threshold must be between 0 and 1", {
   expect_warning(
     winsorize(sample(1:10, 5), threshold = -0.1),
-    regexp = "must be a scalar between 0 and 1"
+    regexp = "must be a scalar between 0 and 0.5"
   )
```

Comment:
It might be that when `insight::format_message()` is used, a line break will by coincident just be inside this string, so matching does no longer work. Maybe just reduce the pattern to `0 and 0.5` or so?

Reply:
Actually this has been solved (essentially, we had modified the warning message within the function but not within the test checks. Harmonizing them fixed it)